design/account.go

```diff
@@ -170,6 +170,7 @@ var userDataAttributes = a.Type("UserDataAttributes", func() {
 	a.Attribute("url", d.String, "The url")
 	a.Attribute("company", d.String, "The company")
 	a.Attribute("providerType", d.String, "The IDP provided this identity")
+	a.Attribute("cluster", d.String, "The OpenShift API URL of the cluster where the user's tenant services are being run")
```

Comment:
"The OpenShift API URL of the cluster where the user is provisioned to"

Reply:
Cool!
24929ff7e472058556a86f06d3db7c9ae886dc24